values.yaml

```
@@ -84,6 +84,12 @@ st2:
   #          enabled: false
 
 
+  # Custom StackStorm config (st2.secret.conf) which will be created from the key 'st2.secret.conf' within this secret.
```

Looks like the file is named `st2.secrets.conf` based on code:
```suggestion
  # Custom StackStorm config (st2.secrets.conf) which will be created from the key 'st2.secrets.conf' within this secret.
```